package.json

```diff
@@ -16,9 +16,9 @@
     "test": "cross-env CONFIG_ENV=dev jest --verbose --config=jest/jest.config.js",
     "test:watch": "npm run test  -- --watch",
     "test:coverage": "npm run test -- --coverage --collectCoverageFrom=**/state/*",
-    "cypress": "cypress",
-    "test:e2e": "cypress run",
-    "test:e2e-chrome": "cypress run --browser chrome",
+    "cypress": "cross-env CONFIG_ENV=staging cypress",
```

Comment:
Why is this important?

Reply:
To configure the staging backend as the base for all endpoints.